libnopegl/src/backends/vk/rendertarget_vk.c

```diff
@@ -87,7 +88,7 @@ static VkResult vk_create_compatible_renderpass(struct gpu_ctx *s, const struct
 
         const VkAttachmentLoadOp load_op   = params ? get_vk_load_op(params->colors[i].load_op)   : VK_ATTACHMENT_LOAD_OP_DONT_CARE;
         const VkAttachmentStoreOp store_op = params ? get_vk_store_op(params->colors[i].store_op) : VK_ATTACHMENT_STORE_OP_DONT_CARE;
-        descs[nb_descs] = (VkAttachmentDescription) {
+        ARRAY_APPEND(descs, (VkAttachmentDescription) {
```

Comment:
I don't find this to be an improvement in readability tbh. I'd rather do `desc[nb_descs++] = ...` so that the two variables are not decorrelated. The switch to `size_t` is good though (and could probably be split out)

Reply:
Switched to size_t in a separate commit.

Reply:
The commit has been dropped.